doc/classes/String.xml

```diff
@@ -43,30 +45,32 @@
 			<return type="bool" />
 			<param index="0" name="text" type="String" />
 			<description>
-				Returns [code]true[/code] if the string begins with the given string.
+				Returns [code]true[/code] if the string begins with the given [param text]. See also [method ends_with].
 			</description>
 		</method>
 		<method name="bigrams" qualifiers="const">
 			<return type="PackedStringArray" />
 			<description>
-				Returns an array containing the bigrams (pairs of consecutive letters) of this string.
+				Returns an array containing the bigrams (pairs of consecutive characters) of this string.
 				[codeblock]
-				print("Bigrams".bigrams()) # Prints ["Bi", "ig", "gr", "ra", "am", "ms"]
+				print("Get up!".bigrams()) # Prints ["Ge", "et", "t ", " u", "up", "p!"]
```

Comment:
I'm curious why the sample text changed, is it to show what would happen with characters such as `!`?